### PR TITLE
Added method to check device biometry

### PR DESCRIPTION
**Pull request by BrunoJAzevedo**
Added checkBiometry method.

Some of the changes are related to version 2.0.1

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -3,17 +3,25 @@
...
     id="outsystems-plugin-fingerprint"
-    version="1.0.1">
+    version="2.0.1">
     <name>fingerprint</name>
     <description>Cordova FingerPrint Plugin</description>
     <license>Apache 2.0</license>
     <keywords>cordova,device</keywords>
-    <dependency id="cordova-plugin-android-fingerprint-auth" url="https://github.com/OutSystems/cordova-plugin-android-fingerprint-auth.git#0.1.os" />
-    <dependency id="cordova-plugin-touchid" url="https://github.com/OutSystems/cordova-plugin-touchid.git#0.4.0.os" />
+    <dependency id="cordova-plugin-android-fingerprint-auth" url="https://github.com/OutSystems/cordova-plugin-android-fingerprint-auth.git#1.4.4" />
```
Comment (Chuckytuh) [inline plugin.xml]: this dependency version does not match the version defined on [package.json](https://github.com/BrunoJAzevedo/cordova-plugin-fingerprint/blob/dev/package.json#L10-L11)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "outsystems-plugin-fingerprint",
+  "version": "2.0.1",
+  "description": "Cordova FingerPrint Plugin",
+  "cordova": {
+    "id": "outsystems-plugin-fingerprint",
+    "platforms": []
+  },
+  "dependencies": {
+    "cordova-plugin-android-fingerprint-auth":"https://github.com/OutSystems/cordova-plugin-android-fingerprint-auth.git#0.3.0-OS",
```
Comment (Chuckytuh) [inline package.json]: I'm not sure if plugin dependencies are to be defined on package.json or not, can't recall an instance where I've seen this happening, usually they are defined on plugin.xml alone 🤔

**Review by Chuckytuh — CHANGES_REQUESTED** [review]

File: www/FingerPrint.js [hunk 3]
```diff
@@ -57,9 +57,23 @@ FingerPrint.prototype.authenticate = function (successCallback, errorCallback,ob
...
     // if the device is ios call the touchid plugin 
     if(cordova.platformId === "ios")touchid.authenticate(successCallback, errorCallback, object.dialogMessage);
 };
 
+FingerPrint.prototype.checkBiometry = function(successCallback, errorCallback) {
+    // success callback function for android 
+    var successCbFingerPrintAuth = function(result) {successCallback(1)};
+    // error callback function for android 
+    var errorCbFingerPrintAuth = function(error) {
+			errorCallback(-1);
```
Comment (Chuckytuh) [inline www/FingerPrint.js:70]: Don't want to nitpick but this is missing style consistency. on line 67 the function is a one-liner, in line 69 it's a multi-line function and the body is not formatted.

**Review by Chuckytuh — CHANGES_REQUESTED** [review]

**Review by Chuckytuh — APPROVED** [review]

**Review by luissilvaos — APPROVED** [review]

**Comment by EiyuuZack** [comment]
Hey @BrunoJAzevedo, could you please change the target of this PR to `OutSystems:master` please?

**Comment by BrunoJAzevedo** [comment]
Hey @EiyuuZack that is done :)

File: www/FingerPrint.js [hunk 4]
```diff
@@ -62,4 +62,18 @@ FingerPrint.prototype.authenticate = function (successCallback, errorCallback,ob
...
+    };
+  
+    // if the device is android call th FingerprintAuth plugin
```
Comment (EiyuuZack) [inline www/FingerPrint.js]: Typo

**Review by EiyuuZack — CHANGES_REQUESTED** [review]
Small typo

File: www/FingerPrint.js (hunk 4 above)
Comment (BrunoJAzevedo) [reply www/FingerPrint.js]: Fixed

**Review by EiyuuZack — APPROVED** [review]

**Pull request merged**